libdevcrypto/Common.cpp

```diff
@@ -104,19 +102,12 @@ Address dev::toAddress(Address const& _from, u256 const& _nonce)
 
 void dev::encrypt(Public const& _k, bytesConstRef _plain, bytes& o_cipher)
 {
-	bytes io = _plain.toBytes();
-	Secp256k1PP::get()->encrypt(_k, io);
-	o_cipher = std::move(io);
+	encryptECIES(_k, _plain, o_cipher);
```

Comment:
`encrypt` & `decrypt` are now the same as `encryptECIES` & `decryptECIES`, do we still need both?